.gitignore

```diff
@@ -40,7 +40,6 @@ node_modules
 
 ### grunt ###
 # Grunt usually compiles files inside this directory
-dist/
```

Comment:
Why is our dist folder being removed from gitignore?

Reply:
I wanted to clean up this file and since we got rid of Grunt, was wondering if the `dist` stuff is Grunt only or serves other purposes as well.

Reply:
Woops, this is actually correct, we use `deploy` not `dist`. It's safe to go away :)